docs/Reference/CLI/Subcommands/Voluntary-Exit.md

```diff
@@ -59,6 +59,39 @@ Endpoint of the beacon node's REST API. Default is `http://127.0.0.1:5051`.
 The path to the YAML configuration file.
 The default is `none`.
 
+### confirmation-enabled
+
+=== "Syntax"
+
+    ```bash
+    teku voluntary-exit --confirmation-enabled=<BOOLEAN>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku voluntary-exit --confirmation-enabled=false
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_CONFIRMATION_ENABLED=false
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    confirmation-enabled: false
+    ```
+
+Specify whether to request confirmation when exiting a validator. Defaults to `true`.
+
+!!! warning
+
+    Exercise care when using this option because if set to `false`, exits are generated immediately
+    without any prompt.
```

Comment:
hrm this might cover my above comment.. i'd like to see neon lights and alarm sirens... can we have sirens? :/

Reply:
Perhaps change the type to "danger"? It will make it red:
![image](https://user-images.githubusercontent.com/4677568/101153580-ed20bb80-3624-11eb-909e-d052691f1d3d.png)

Reply:
i actually kinda like highlighting it like that, @ajsutton ?